website/src/components/discourse/index.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import feedStyles from './styles.module.css';
+
+export default function DiscourseFeed({
+  order = 'latest_topic',
+  status = 'solved',
+  after = undefined,
+  before = undefined,
+  inString = undefined, 
+  min_posts = undefined,
+  max_posts = undefined,
+  min_views = undefined,
+  max_views = undefined,
+  tags = undefined, 
+  term = undefined,
+  category = 'help',
+  title = undefined,
+  link_text = "Ask the Community",
+  link_href = `https://discourse.getdbt.com/new-topic${category ? `?category=${category}` : ''}${tags ? (!category ? `?tags=${tags}` : `&tags=${tags}`) : ''}`,
+  hide_cta = false,
```

Comment:
Isn't it more standard to have positive flags, so it'd be `show_cta = true`? Non-blocking